accepted/0000-yarn-global-run

```diff
@@ -0,0 +1,43 @@
+- Start Date: 2017-09-12
+- RFC PR: (leave this empty)
+- Yarn Issue: [yarnpkg/yarn#4392](https://github.com/yarnpkg/yarn/issues/4392)
+
+# Summary
+
+Adding a `yarn global run` command, which is a more generic version of 
+`yarn create`. It will globally install the package and run its binary with the
+arguments that are passed after `yarn global run`.
+
+# Motivation
+
+Some packages that are similar in design to `create-react-app` ([vue-cli](https://yarn.pm/vue-cli),
+[preact-cli](https://yarn.pm/preact-cli), [@angular/cli](https://yarn.pm/@angular/cli)
+...) don't have a name that's prefixed with `create-`, and thus prevent the usage of
+`yarn create`, even though it would be perfectly suitable for it. 
+
+# Detailed design
+
+The design would be exactly the same as `yarn create`, but without the prefixing of
+`create-`. 
+
+# How We Teach This
+
+Yarn run and Yarn global already exist, so this is not introducing new concepts, 
+especially since `yarn create` already exists.
+
+# Drawbacks
+
+It can cause confusion with regard to `yarn create`, since it has a very similar
```

Comment:
May be we can come up with some safety measure for `yarn create` that limits damage/impact for scripts adhering to certain standards and `yarn global run` becomes the "escape hatch" everything else?

One thing that I can think of is not making binaries from `yarn create` installs globally available at all so you __have to__ run `yarn create react-app` even after installing it.